src/shared/ui/select/Select.tsx

```
@@ -0,0 +1,106 @@
+        className={twMerge(
+          'max-h-[300px] w-full overflow-y-auto absolute top-11 flex flex-col bg-gray-800 border border-gray-700 rounded  z-10 transition-all duration-200 origin-top ',
+          dropdown ? 'opacity-100 scale-100' : 'opacity-0 scale-95 pointer-events-none'
+        )}
+      >
+        <div
+          className="flex flex-row gap-2 items-center px-1 hover:bg-white hover:text-black cursor-pointer py-1 rounded "
+          onClick={() => {
+            setValue('');
+            setDropdown(false);
+          }}
+        >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)